pages/ksphere/kommander/1.2/install-airgapped/index.md

```diff
@@ -29,11 +29,11 @@ Before installing, ensure your environment has the following basic requirements:
   - management cluster must connect to the attached cluster's API server
   - management cluster must connect to load balancers created by some addons. For example, Thanos, part of the Prometheus addon, connects to those load balancers.
 
-- all the prerequisites in [air-gapped Konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#before-you-begin] in case of Konvoy clusters. For attached 3rd party clusters refer to the vendor's documentation.
```

Comment:
Why do we want to remove this?

Reply:
IMO this doesn't have any value because what could a 3rd party vendor have to say about the prerequisites of attaching its cluster to Kommander? I guess nothing.

Reply:
For me, it is more of a statement of delimiting responsibility: "Hey - we can tell you what Konvoy needs, but 3rd party clusters are not our responsibility and you need to take care of it yourself.

Reply:
But how should they take care of it? There's no way for us to provide any action specific to satisfying prerequisites because we don't know what the prerequisites are for non-Konvoy clusters, right? So what we could say is "hey, we cannot tell you the prerequisites for your 3rd-party cluster and you may just have to find them out yourself by trial and error" which IMHO is not a very positive messaging to confront the customer with.

Reply:
OK, I see your point.